src/commands/protocols/update/receiver/v1.0.0/v1-0-0-handle-update-init-command.js

```diff
@@ -0,0 +1,86 @@
+import HandleProtocolMessageCommand from '../../../common/handle-protocol-message-command.js';
+import { ERROR_TYPE, OPERATION_ID_STATUS } from '../../../../../constants/constants.js';
+
+class HandleUpdateInitCommand extends HandleProtocolMessageCommand {
+    constructor(ctx) {
+        super(ctx);
+        this.ualService = ctx.ualService;
+        this.shardingTableService = ctx.shardingTableService;
+        this.blockchainModuleManager = ctx.blockchainModuleManager;
+        this.serviceAgreementService = ctx.serviceAgreementService;
+        this.repositoryModuleManager = ctx.repositoryModuleManager;
+
+        this.errorType = ERROR_TYPE.UPDATE.UPDATE_REMOTE_ERROR;
+    }
+
+    async prepareMessage(commandData) {
+        const { operationId, assertionId, blockchain, contract, tokenId, keyword, hashFunctionId } =
+            commandData;
+        await this.operationIdService.updateOperationIdStatus(
+            operationId,
+            OPERATION_ID_STATUS.VALIDATE_ASSET_REMOTE_START,
+        );
+
+        await this.operationIdService.cacheOperationIdData(operationId, {
+            assertionId,
+            blockchain,
+            contract,
+            tokenId,
+            keyword,
+            hashFunctionId,
+        });
+
+        const validationResult = await this.validateReceivedData(
+            operationId,
+            assertionId,
+            blockchain,
+            contract,
+            tokenId,
+            keyword,
+            hashFunctionId,
+        );
+
+        await this.operationIdService.updateOperationIdStatus(
+            operationId,
+            OPERATION_ID_STATUS.VALIDATE_ASSET_REMOTE_END,
+        );
+        return validationResult;
+    }
+
+    async validateAssertionId(blockchain, contract, tokenId, assertionId, ual) {
```

Comment:
Can't we make a universal function for `HandleProtocolMessageCommand` in order not to overwrite function here?

Reply:
But the method is different that's the reason why we do overwrite